core-bundle/assets/controllers/deeplink-controller.js

```diff
@@ -0,0 +1,70 @@
+import { Controller } from '@hotwired/stimulus';
+import * as Turbo from '@hotwired/turbo';
+
+export default class extends Controller {
+    static targets = ['primary', 'secondary'];
+
+    static afterLoad (identifier, application) {
+        const setupController = () => {
+            document.querySelectorAll('.click2edit').forEach((el) => {
+                el.classList.remove('click2edit');
+
+                const primary = el.querySelector('a.edit');
+                const secondary = el.querySelector('a.children');
+
+                if (primary) {
+                    primary.setAttribute(`data-${identifier}-target`, "primary");
+                }
+
+                if (secondary) {
+                    secondary.setAttribute(`data-${identifier}-target`, "secondary");
+                }
+
+                el.dataset.controller = `${el.dataset.controller || ''} ${identifier}`;
+            });
+        };
+
+        document.addEventListener('DOMContentLoaded', setupController);
+        document.addEventListener('ajax_change', setupController);
+        document.addEventListener('turbo:render', setupController);
+        document.addEventListener('turbo:frame-render', setupController);
+        setupController();
+
+        Theme.setupCtrlClick = () => {
+            console.warn('Using Theme.setupCtrlClick() is deprecated and will be removed in Contao 6. Apply the Stimulus actions instead.');
+            setupController();
+        }
+    }
+
+    initialize () {
+        this.visitPrimary = this.visitPrimary.bind(this);
+        this.visitSecondary = this.visitSecondary.bind(this);
+    }
+
+    connect () {
+        this.element.addEventListener('click', this.visitPrimary);
+        this.element.addEventListener('click', this.visitSecondary);
+    }
+
+    disconnect () {
+        this.element.removeEventListener('click', this.visitPrimary);
+        this.element.removeEventListener('click', this.visitSecondary);
+    }
+
+    visitPrimary (event) {
+        const key = window.navigator.platform?.startsWith('Mac') ? 'metaKey' : 'ctrlKey';
```

Comment:
I would prefer to use native Stimulus events here (`data-action="click.alt->contao--deeplink#visitPrimary`), but unfortunately there is no way to have a different key modifier per platform. Implementing our own seems kinda overkill too.

Reply:
Turbo seems to support

```html
data-action="alt+click->contao--deplink#visitPrimary"
```

which will automatically be the option key on MacOS.

https://github.com/hotwired/stimulus/pull/665
https://stimulus.hotwired.dev/reference/actions#keyboardevent-filter

Reply:
yes, but we are using meta key on mac and ctrl key on windows

Reply:
Maybe we can unify that in a separate PR?

Reply:
What do you mean by "unify"?

Reply:
Sounds good to me. Or generally rethink some "quick edit" stuff when reworking the operations (context menu etc.). I'd leave this as it is here then.